autoload/neomake/makers/ft/d.vim

```diff
@@ -45,15 +45,15 @@ function! s:DmdStyleMaker(args)
     call s:UpdateDub()
     let l:args = ['-w', '-wi', '-c', '-o-', '-vcolumns'] + a:args + s:dubImports
     return {
-        \ 'args': l:args,
-        \ 'errorformat':
-        \     '%f(%l\,%c): %trror: %m,' .
-        \     '%f(%l): %trror: %m,' .
-        \     '%f(%l\,%c): %tarning: %m,' .
-        \     '%f(%l): %tarning: %m,' .
-        \     '%f(%l\,%c): Deprecation: %m,' .
-        \     '%f(%l): Deprecation: %m,',
-        \ }
+                \ 'args': l:args,
+                \ 'errorformat':
+                \     '%f(%l\,%c): %trror: %m,' .
+                \     '%f(%l): %trror: %m,' .
+                \     '%f(%l\,%c): %tarning: %m,' .
+                \     '%f(%l): %tarning: %m,' .
+                \     '%f(%l\,%c): Deprecation: %m,' .
+                \     '%f(%l): Deprecation: %m,',
+                \ }
```

Comment:
I prefer this style, where there is inline spacing to show that the following string is the value of the above key